### DISCUSSION TITLE
Installation Guide

### DISCUSSION THREAD

**Discussion by TheuDeXter** [stustapay/stustapay#261] [Q&A]
Hello, congratulations on the project!
Do you have any installation guide?
Thanks

**Comment by TheJJ** [comment]
We don't have a full guide yet, but the components and their usage are described in the docs. 

If you are familiar with software development best practices, you should be able to set up the api servers and web frontends, build the android app and set up web servers. The TSE setup is more tricky, you basically need a Diebold TSE usb stick.

What's your use-case for the payment system?

**Comment by TheuDeXter** [comment]
> We don't have a full guide yet, but the components and their usage are described in the docs.
> 
> If you are familiar with software development best practices, you should be able to set up the api servers and web frontends, build the android app and set up web servers. The TSE setup is more tricky, you basically need a Diebold TSE usb stick.
> 
> What's your use-case for the payment system?

I need this system to use in my bar.
I don't know much about this language, I needed an installation guide.
Are you interested in carrying out freelance installation? We combine values.
Hugs!!

**Comment by TheJJ** [comment]
Hey! I think the first hurdle you need to pass is have a linux server where the service is running and the nfc cards with the correct programming :) on server you need to have a postgres database and the terminalapi, the adminui + api, receipt generation, and depending on your legislation the TSE service. If you want online topup/payout the customer portal + api. Then you need android phones with a proper nfc reader, and the deploy the app on them. Have a stable wifi (preferrably exclusive for the payments) where the phones can reach the terminalserver api.
Quite some challenges, but some you need to solve for any other payment system - but i'm curious how your bar project turns out :)

**Comment by CREO-IT** [comment]
the docs mention : https://github.com/stustapay/stustapay/releases

but there s nothing. ?
